src/shared/services/order/order.service.ts

```diff
@@ -27,6 +27,23 @@ export class OrderService {
     return a.order - b.order;
   };
 
+  public static sortListItems(caseField: CaseField) {
```

Comment:
Can we add tests for cases:
1) we do not have order values specified - final order should reverse the original order of elements
2) order is provided but some elements are missing the order - final order should reverse the original order of elements
3) order is provided for all elements - final order should follow the defined order

Reply:
As discussed I understand that 2. is not to be covered by this PR as there will be future validation logic added in def store that will stop users from importing this partial state.
1. and 3. are covered as part of write-fixed-list-field.component.spec.ts.

These tests should really be moved to order.service.ts at some point but a minor comment really.